include/unifex/submit.hpp

```diff
@@ -26,9 +26,18 @@
 #include <unifex/scope_guard.hpp>
 #include <unifex/std_concepts.hpp>
 
+#include <memory>
+
 #include <unifex/detail/prologue.hpp>
 
 namespace unifex {
+template <typename Allocator, typename T>
+using rebind_alloc_t =
```

Comment:
Should these type-aliases be in a detail namespace?

Reply:
Yes, probably.